ci/deps/actions-311.yaml

```diff
@@ -0,0 +1,56 @@
+name: pandas-dev-test
+channels:
+  - conda-forge
+dependencies:
+  - python=3.11
+
+  # build dependencies
+  - versioneer[toml]
+  - cython>=0.29.32
+
+  # test dependencies
+  - pytest>=6.0
```

Comment:
Noting some of these got updated in https://github.com/pandas-dev/pandas/pull/50481

Reply:
Adjusted